tests/contrib/botocore/test.py

```diff
@@ -3765,3 +3774,259 @@ def test_schematized_unspecified_service_secretsmanager_v1(self):
 
             assert span.service == DEFAULT_SPAN_SERVICE_NAME
             assert span.name == "aws.secretsmanager.request"
+
+    @pytest.mark.snapshot(ignores=snapshot_ignores)
+    @mock_sqs
+    def test_aws_payload_tagging_sqs(self):
+        with self.override_config("botocore", dict(payload_tagging_request="all", payload_tagging_response="all")):
```

Comment:
Could we also add test cases for when `payload_tagging_request` and/or `payload_tagging_response` is not equal to `all`?